examples/tutorials/api/nested_sampling_Crab.py

```diff
@@ -0,0 +1,399 @@
+# used for model comparison.
+# For more information see : `on the use of the evidence for model comparison
+# <https://ned.ipac.caltech.edu/level5/Sept13/Trotta/Trotta4.html>`__.
+#
+# **Results stored on disk**
+#
+# if `log_dir` is set to a name where the results will be stored, then
+# a directory is created containing many useful results and plots.
+# A description of these outputes is given `here in Ultranest
+# docs <https://johannesbuchner.github.io/UltraNest/performance.html#output-files>`__.
+#
+
+
+######################################################################
+# Results
+# -------
+#
+
+
+######################################################################
+# Within a Bayesian analysis, the concept of best-fit has to be viewed
+# differently from what is done in a gradient descent fit.
+#
+# The output of the Bayesian analysis is the posterior distribution and
+# there is no “best-fit” output.
+# One has to define, based on the posteriors, what we want to consider
+# as “best-fit” and several options are possible:
+#
+# -  the mean of the distribution
+# -  the median
+# -  the lowest likelihood value
+#
+# By default the `~gammapy.modeling.models.DatasetModels` will be updated with the `mean` of
+# the posterior distributions.
+#
+
+result_joint.models
+
+
+######################################################################
+# The Sampler class returns a very rich dictionnary.
+# The most “standard” information about the posterior distributions can
+# be found in :
+#
+
+result_joint.sampler_results["posterior"]
+
+
+######################################################################
+# Besides mean, errors, etc, an interesting value is the
+# `information gain` which estimates how much the posterior
+# distribution has shrinked with respect to the prior (i.e. how much
+# we’ve learned). A value < 1 means that the parameter is poorly
+# constrained with the prior range.
+# For a more physical interpretation see this
+# `example <https://arxiv.org/abs/2205.00009>`__.
+#
+# The `Sampler Result` dictionnary contains also other interesting
+# information :
+#
+
+result_joint.sampler_results.keys()
+
+
+######################################################################
+# Of particular interest, the samples used in the process to approximate
+# the posterior distribution can be accessed via :
+#
+
+for i, n in enumerate(model.parameters.free_parameters.names):
+    s = result_joint.samples[:, i]
+    fig, ax = plt.subplots()
+    ax.hist(s, bins=30)
+    ax.axvline(np.mean(s), ls="--", color="red")
+    ax.set_xlabel(n)
+    plt.show()
+
+
+######################################################################
+# While the above plots are interesting, the real strength of the Bayesian
+# analysis is to visualise all parameters correlations which is usually
+# done using “corner plots”.
+#
+
+from ultranest.plot import cornerplot
+
+cornerplot(result_joint.sampler_results, plot_datapoints=True, plot_density=True)
```

Comment:
You can encourage users here to make their own custom corner plot, by copying the coder from ultranest's cornerplot() function here. There are many packages and every one has their favorite!

Reply:
I added some links to the API docs. Not the full code to avoid overloading.
The `title_fmt` corner kw is what I was looking for a long time (the amplitude of 4e-11 was shown as 0.00 before).